client.go

```diff
@@ -38,6 +38,8 @@ type NewClientInput struct {
 	AuthenticationMethod AuthenticationMethod
 	OAuthToken           string
 	OAuthTokenSecret     string
+	ApiKey               string
+	ApiKeySecret         string
```

Comment:
@EricFrancis12 
Thank you for the useful additions!
Your implementation is perfect, including the test code, but I would like to make one correction.

In this library, the keyword “API” is written as `api` or `API` in the code. Therefore, I would like to ask you to also refer to the newly added `ApiKey` and `ApiKeySecret` as `APIKey` and `APIKeySecret`.

Reply:
I have confirmed that it has been corrected.
Thank you for your response.